tests/acceptance/test_checkbox.py

```diff
@@ -82,6 +82,41 @@ def test_out_of_bounds_down(self):
         self.sut.expect(r"{'interests': \['Books'\]}.*", timeout=1)  # noqa
 
 
+@unittest.skipUnless(sys.platform.startswith("lin"), "Linux only")
+class CheckOtherTest(unittest.TestCase):
+    def setUp(self):
+        self.sut = pexpect.spawn("python examples/checkbox_other.py")
+        self.sut.expect("Computers.*", timeout=1)
+
+    def test_other_input(self):
+        self.sut.send(key.UP)
+        self.sut.expect(r"\+ Other\.\.\..*", timeout=1)
+        self.sut.send(key.SPACE)
+        self.sut.expect(r": ", timeout=1)
+        self.sut.send("Hello world")
+        self.sut.expect(r"Hello world.*", timeout=1)
+        self.sut.send(key.ENTER)
+        self.sut.expect(r"> X Hello world[\s\S]*\+ Other\.\.\..*", timeout=1)
```

Comment:
the tests are still checking it there are ... aftrr Other. This is why it is failing.